RetrospectiveExtension.Frontend/components/feedbackItem.tsx

```diff
@@ -977,3 +963,26 @@ export default class FeedbackItem extends React.Component<IFeedbackItemProps, IF
     );
   }
 }
+
+export class FeedbackItemHelper
+{
+// Handle linking/grouping workitems and reload any updated items.
+public static handleDropFeedbackItemOnFeedbackItem = async (feedbackItemProps: IFeedbackItemProps, droppedItemId: string, targetItemId: string) => {
```

Comment:
nit: something wrong with the indentation for this method

Reply:
fixed indentation